packages/ses/docs/lockdown.md

```diff
@@ -366,12 +367,54 @@ the container to exit explicitly, and we highly recommend setting
 - `'none'`: do not install traps for uncaught exceptions. Errors are likely to
   appear as `{}` when they are reported by the default trap.
 
+## `evalTaming` Options
+
+This option only affects the initial compartment!
+
+**Background**: For every compartment including the implied compartment in the
+initial JavaScript realm, there are evaluators `eval` and `Function`.
+The default lockdown behavior isolates all of these evaluators.
+
+Replacing the realm's initial evaluators is not necessary to ensure the
+isolation of guest code because guest code must not run in the initial realm.
+However, replacing these evaluators is the safest and therefore default
+evaluator taming option (`"safeEval"`) because it may limit the harm that guest code can cause if it escapes its assigned compartment.
+
+However, only the exact `eval` function from the initial realm can be used to
```

Comment:
```suggestion
However, in the shim, only the exact `eval` function from the start compartment can be used to
```
I added "in the shim" because the hardened JS proposal and (IIUC) the XS implementation have no such restriction on direct eval.